### ISSUE TITLE
iOS crash on startup

### ISSUE THREAD

**Issue by vanlooverenkoen** [BeMacized/xiaomi_scale#3]
I copied the example dart code to my project and the app crashes on startup.

**Comment by vanlooverenkoen** [comment]
I forgot to add the background capability & added

***OUTDATED***

```
	<key>NSBluetoothAlwaysUsageDescription</key>
	<string>The app uses bluetooth to find, connect and transfer data between different devices</string>
	<key>NSBluetoothPeripheralUsageDescription</key>
	<string>The app uses bluetooth to find, connect and transfer data between different devices</string>
	<key>UIBackgroundModes</key>
	<array>
		<string>bluetooth-central</string>
	</array>
```
![image](https://user-images.githubusercontent.com/21172855/88100963-9fd40e00-cb9d-11ea-9044-159f6f02a47b.png)

correct answer (2 comments down) -> https://github.com/BeMacized/xiaomi_scale/issues/3#issuecomment-662662144

**Comment by vanlooverenkoen** [comment]
#5 documentation should be added

**Comment by vanlooverenkoen** [comment]
After doing some research today (resolved in a pr #6 ). I found ot that the UIBackgroudModes are not needed. The 2 descriptions are.

This is what you should add to your info.plist (customize your text for your needs)

```
	<key>NSBluetoothAlwaysUsageDescription</key>
	<string>The app uses bluetooth to find, connect and transfer data between different devices</string>
	<key>NSBluetoothPeripheralUsageDescription</key>
	<string>The app uses bluetooth to find, connect and transfer data between different devices</string>
```
